src/data/redis-cache/__tests__/data-source.tests.js

```diff
@@ -0,0 +1,72 @@
+import CacheDS from '../data-source';
+
+describe('the matrix item data source', () => {
```

Comment:
Is this a copy/paste hiccup?

```suggestion
describe('the redis cache data source', () => {
```